src/Microsoft.Compatibility.HashCode/src/Configurations.props

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <PropertyGroup>
+    <PackageConfigurations>
+      netstandard;
+      netstandard2.1;
+      netcoreapp2.1;
+      uap;
```

Comment:
Same comments as ref I believe you can remove uap, change netstandard to netstandard2.0

Reply:
netstandard is already netstandard2.0, I didn't repourpose the versionless to now mean netstandard2.1. Do we still want to change it here?

Reply:
I'm just reccomending we avoid using versionless targetgroups in packages.   Just use the versioned targetgroups.  It creates less stuff for us to clean up in the future when we need to move the versionless target groups to mean a new version.